### PR TITLE
Latest generation

### PR DESCRIPTION
**Pull request by ahuang11**
## Summary

Adds useful return docstrings and also an updated schema.

## Checklist
- [ ] Summarized PR's changes in [CHANGELOG.md](https://github.com/PrefectHQ/prefect-databricks/blob/main/CHANGELOG.md)

### CODE REVIEW

**Comment by ahuang11** [comment]
image
image
image

**Comment by ahuang11** [comment]
This is ready now.

File: prefect_databricks/flows.py [hunk 1]
```diff
@@ -52,8 +52,9 @@ class DatabricksJobRunTimedOut(Exception):
 
 @flow(
     name="Submit jobs runs and wait for completion",
-    description="Triggers a Databricks jobs runs and waits for the"
-    "triggered runs to complete.",
+    description=(
+        "Triggers a Databricks jobs runs and waits for thetriggered runs to complete."
```
Comment (desertaxle) [inline prefect_databricks/flows.py:56]: We're missing a space here. Is this typo coming for the OpenAPI spec?

**Review by desertaxle — COMMENTED** [review]
Do we know if there are any breaking changes in this PR?

**Review by desertaxle — APPROVED** [review]

**Review by desertaxle — APPROVED** [review]

**Pull request merged**